### ISSUE TITLE
Rocket launcher losing all ammo upon save load

### ISSUE THREAD

**Issue by vekst** [EvaisaDev/LethalThings#33]
Seems to have started happening with the latest update to the mod. Previously if you loaded a save while owning a rocket launcher, it would give it full ammo, while now after the latest update it removes all ammo from it.

**Comment by Dear-Dee** [comment]
Yes, I also encountered the same situation. Every time we go offline and then go online again, regardless of whether there is any remaining ammunition, its ammunition will be reset to zero.
